pkg/sync/syncer.go

```
@@ -955,19 +865,105 @@
 			if dontFixCycles {
 				return fmt.Errorf("cycles detected in entitlement graph")
 			}
-
+			fmt.Printf("fixing cycles1\n")
 			err := entitlementGraph.FixCycles()
+			fmt.Printf("fixed cycles1\n")
```

golangci-lint (1.62.2)

use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)